selftests/unit/test_runner_queue.py

```diff
@@ -42,12 +43,15 @@ def test_whiteboard(self):
         """
         Tests if the whiteboard content is the expected one
         """
+        this = os.path.abspath(__file__)
+        base = os.path.dirname(os.path.dirname(os.path.dirname(this)))
+        module = os.path.join(base, 'examples', 'tests', 'whiteboard.py')
         factory = ['WhiteBoard',
                    {'methodName': 'test',
                     'tags': set([]),
                     'params': ([TreeNode(name='')], ['/run/*']),
                     'job': self.job,
-                    'modulePath': 'examples/tests/whiteboard.py',
+                    'modulePath': module,
```

Comment:
oh, ok. Maybe we should start validating our new code against Py3 as well.

Reply:
Yep, that's the goal of this PR.  Since it's a pain to run "make check" on both versions locally, let's use Travis for that.